src/app/users/get-user.ts

```diff
@@ -1,65 +1,53 @@
-import {
-  SemanticAttributes,
-  asyncRunInSpan,
-  addAttributesToCurrentSpan,
-  ENDUSER_ALIAS,
-} from "@services/tracing"
+import { SemanticAttributes, asyncRunInSpan } from "@services/tracing"
 import { UsersRepository } from "@services/mongoose"
-import { IpFetcher } from "@services/ipfetcher"
 import { getIpConfig } from "@config/app"
-import { RepositoryError } from "@domain/errors"
+import { IpFetcher } from "@services/ipfetcher"
 
 const users = UsersRepository()
 
-export const getUserForLogin = async ({
+export const updateIpInfo = async ({
   userId,
+  iPs,
   ip,
+  lastConnection,
 }: {
-  userId: string
-  ip?: string
-}): Promise<User | ApplicationError> =>
+  userId: UserId
+  iPs: IPType[]
+  ip?: Ip
+  lastConnection: Date
+}): Promise<void> =>
   asyncRunInSpan(
     "app.getUserForLogin",
-    { [SemanticAttributes.CODE_FUNCTION]: "getUserForLogin" },
+    { [SemanticAttributes.CODE_FUNCTION]: "addIp" },
```

Comment:
If you want to change this you should also change the name of the span `app.getUserForLogin`.